redaxo/src/addons/media_manager/lib/effects/effect_convert2img.php

```diff
@@ -19,6 +19,7 @@ class rex_effect_convert2img extends rex_effect_abstract
         'bmp',
         'eps',
         'ico',
+        'webp',
```

Comment:
Ist das hier sonnvoll, wenn wir doch webp nativ können?
jpg, png und gif sind ja auch nicht hier in der liste..